equalla-web-components/src/components/SearchResults.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <div class="search-results">
+    <h1>{{ msg }}</h1>
+    <p>
+      For guide and recipes on how to configure / customize this project,<br>
+      check out the
+      <a href="https://github.com/vuejs/vue-cli/tree/dev/docs" target="_blank">vue-cli documentation</a>.
+    </p>
+    <ul>
+      <li v-for="item in items" :key="item.uuid">
+        <a v-bind:href="item.url">{{ item.name }}</a>
+        <p>{{ item.description }}</p>
+      </li>
+    </ul>
+  </div>
+</template>
+
+<script>
+export default {
+  props: {
+    msg: String
+  },
+  data() {
+    return {
+      items: []
+    }
+  },
+  async created() {
+    try {
+      const response = await fetch('/sample/search.json');
+      if (!response.ok) {
+        throw new Error(response.statusText);
+      }
+      const data = await response.json();
+      for (const item of data) {
```

Comment:
consider using `this.items = data.map(item => {...})`.
this will ensure that vue re-render will always be called after all the data has been rangled.